src/nifax/control/controller/TableGralController.java

```diff
@@ -0,0 +1,179 @@
+package nifax.control.controller;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Vector;
+import javax.swing.JOptionPane;
+import javax.swing.table.DefaultTableModel;
+import nifax.control.data.MapDb;
+import nifax.control.model.Category;
+import nifax.control.model.Measure;
+import nifax.control.model.Price;
+import nifax.control.model.Store;
+import nifax.control.model.modeler.CategoryOperation;
+import nifax.control.model.modeler.HQLOperation;
+import nifax.control.model.modeler.MeasureOperation;
+import nifax.control.model.modeler.PriceOperation;
+import nifax.control.model.modeler.StoreOperation;
+import nifax.control.util.ColumnSorter;
+import nifax.control.util.Table;
+import nifax.control.view.panel.PanelGeneralAdmin;
+
+/**
+ *
+ * @author NB
+ */
+public final class TableGralController {
+
+    private static TableGralController instance = null;
+
+    private PanelGeneralAdmin panelGeneralAdmin;
+
+    public static final String Category = "Categoria";
+    public static final String Store = "Deposito";
+    public static final String Price = "Lista de precio";
+    public static final String Measure = "Medida";
+
+    protected TableGralController() {
+    }
+
+    public static TableGralController getInstance() {
+        if (instance == null) {
+            instance = new TableGralController();
+        }
+        return instance;
+    }
+
+    public void setPanelGeneralAdmin(PanelGeneralAdmin panelGeneralAdmin) {
+        this.panelGeneralAdmin = panelGeneralAdmin;
+    }
+
+    public void list(String titleTab) {
+        Table.clear(panelGeneralAdmin.getTbl_gral());
+
+        switch (titleTab) {
```

Comment:
every case on this switch looks very similar, I think you can avoid duplicate code